components/label/Label.js

```diff
@@ -0,0 +1,67 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Input from '../input';
+import Box from '../box';
+import { TextBody, TextDisplay, TextSmall } from '../typography';
+import theme from './theme.css';
+import isComponentOfType from '../utils/is-component-of-type';
+import cx from 'classnames';
+
+export default class Label extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    children: PropTypes.oneOfType([PropTypes.element, PropTypes.string, PropTypes.array]),
+    for: PropTypes.string,
+    inverse: PropTypes.bool,
+    helpText: PropTypes.string,
+    required: PropTypes.bool,
+    size: PropTypes.oneOf(['small', 'medium', 'large']),
+  };
+
+  static defaultProps = {
+    inverse: false,
+    helpText: 'Optional',
```

Comment:
I guess the default should be empty/false?

Reply:
When adding `required` to the Label component helpText is removed. In some way I think it's logical, but then again I also think it's weird because the label is not required, but the input field... However, I think it would be way too hard to check the required prop on the Input component to change things in the Label component. 

In a way I think having it default empty is more logical, but then again, it also reminds you to specify if an input field is required or not...

I'm just listing pro's and cons here 😄